jsapp/js/account/security/password/updatePasswordForm.component.tsx

```diff
@@ -93,6 +95,15 @@ export default function UpdatePasswordForm() {
 
   return (
     <form className={styles.root} onSubmit={submitPasswordForm}>
+      {envStore.data.enable_custom_password_guidance_text && (
```

Comment:
If the `envStore` isn't ready the first time this renders, the guidance text doesn't show up until the user starts typing (triggering a state change). Could be fixed by checking if `envStore.isReady` is true.

Reply:
I pushed a fix that uses `useEffect` :)

Reply:
@LMNTL that is educational, thanks :) Pushed a final fix 😉